justfile

```diff
@@ -73,6 +76,26 @@ check-links: generate-docs
     cargo deadlinks --dir target/doc/embedded_graphics
     cargo deadlinks --dir target/doc/embedded_graphics_core
 
+# Generate drawing examples in the doc directory
+generate-drawing-examples:
+    cd tools/generate-drawing-examples && cargo run
+
+# Checks if drawing examples are up to date
+check-drawing-examples: generate-drawing-examples
+    git diff --quiet doc/ || ( \
+        echo "doc/ folder is not up to date" \
+        echo "Try running 'just generate-drawing-examles'." \
+        echo "If any images have changed, run just generate-drawing-examples-montage' to update the collage image too" \
```

Comment:
```suggestion
        echo "If any images have changed, run 'just generate-drawing-examples-montage' to update the collage image too" \
```

Reply:
Good catch! This suggestion also made me notice a typo in the line above.